adapter/thing_management.md

```diff
@@ -0,0 +1,404 @@
+# Thing Management
+
+The following document describes the interaction flows and FIMP specification used in management of things.
+
+## Flows
+
+### Network Inclusion
```

Comment:
In Z-Wave specification to hardly differentiate Smart Start Inclusion with "normal" Inclusion processes, they call it a "Classic Inclusion"/Classic Exclusion. Maybe we can also apply that?